starter-core/src/main/java/io/micronaut/starter/feature/build/gradle/templates/testFrameworks.rocker.raw

```diff
@@ -35,7 +35,7 @@ TestFeature testFeature
     @dependency.template("org.junit.jupiter", "junit-jupiter-api", "testImplementation", null)
     @dependency.template("io.micronaut.test", "micronaut-test-junit5", "testImplementation", null)
     @dependency.template("org.junit.jupiter", "junit-jupiter-engine", "testRuntimeOnly", null)
-} else if (testFeature.isKotlinTest()) {
+} else if (testFeature.isKotlinTest() || testFeature.isKoTest()) {
```

Comment:
This could be simpler if this PR was merged: https://github.com/micronaut-projects/micronaut-starter/pull/411